addons/website_slides/models/slide_channel.py

```diff
@@ -541,7 +541,8 @@ def _filter_add_members(self, target_partners, **member_values):
                 on_invite.check_access_rights('write')
                 on_invite.check_access_rule('write')
             except:
```

Comment:
Actually, it doesn't look too good, but if we want to avoid inducing other changes, perhaps we should do something like this :thinking: 
```suggestion
            except AccessError:
                if key in context:
                    raise AccessError(_("..."))  # not an emoji
            except:
                pass
```

Reply:
I think we only want to pass in case of AccessError anyway, so we probably don't need your last part (except pass).

In case of other errors, then it's fine to get a traceback.

Reply:
Updated :+1: